csharp/extractor/Semmle.Extraction/Context.cs

```diff
@@ -225,14 +225,6 @@ public void PopulateAll()
         /// </summary>
         public Compilation Compilation { get; }
 
-        /// <summary>
-        /// Create a new context, one per source file/assembly.
-        /// </summary>
-        /// <param name="e">The extractor.</param>
-        /// <param name="c">The Roslyn compilation.</param>
-        /// <param name="extractedEntity">Name of the source/dll file.</param>
-        /// <param name="scope">Defines which symbols are included in the trap file (e.g. AssemblyScope or SourceScope)</param>
-        /// <param name="addAssemblyTrapPrefix">Whether to add assembly prefixes to TRAP labels.</param>
         public Context(IExtractor e, Compilation c, TrapWriter trapWriter, IExtractionScope scope, bool addAssemblyTrapPrefix)
```

Comment:
Why did you remove the doc comments?

Reply:
Because it was not matching the code. Also, one of the main goals is to refactor/split the `Context` class, so I expect more changes, which would make the comment even more outdated.

Reply:
OK